de4a-connector/src/main/java/eu/de4a/connector/as4/client/regrep/RegRepTransformer.java

```diff
@@ -63,24 +63,31 @@ public Element wrapMessage(Element canonical, boolean request) throws MessageExc
 			Source src = new DOMSource();
 			OutputStream xmlFile = new ByteArrayOutputStream();
 			Transformer transformerxsl = factory.newTransformer(xslDoc);
-			fillParameter(transformerxsl, CONSUMER_ID, DOMUtils.getValueFromXpath(XPATH_CONSUMER_ID, canonical));
-			fillParameter(transformerxsl, CONSUMER_NAME, DOMUtils.getValueFromXpath(XPATH_CONSUMER_NAME, canonical));
-			fillParameter(transformerxsl, PERSONAL_ID, DOMUtils.getValueFromXpath(XPATH_PERSONAL_ID, canonical));
-			fillParameter(transformerxsl, CURRENT_TIME, getCurrentTime());
-			fillParameter(transformerxsl, PROVIDER_ID, DOMUtils.getValueFromXpath(XPATH_PROVIDER_ID, canonical));
-			fillParameter(transformerxsl, PROVIDER_NAME, DOMUtils.getValueFromXpath(XPATH_PROVIDER_NAME, canonical));
+
```

Comment:
Are these changes really worth the effort?

Reply:
They were done before we discuss about the ridding of Toop, so, meanwhile we could make do with this...